ably-ios/ARTTypes.h

```diff
@@ -47,9 +47,7 @@ typedef NS_ENUM(NSUInteger, ARTRealtimeChannelState) {
     ARTRealtimeChannelInitialised,
     ARTRealtimeChannelAttaching,
     ARTRealtimeChannelAttached,
-    ARTRealtimeChannelDetaching,
```

Comment:
I changed it according to http://docs.ably.io/realtime/channels-messages/ _Channel states_ topic. I will undo.